ct.sh

```diff
@@ -23,6 +24,7 @@ main() {
     local config=
     local command=
     local kubeconfig="$HOME/.kube/config"
+    local dockerArgs=()
```

Comment:
Shell scripts need snake case. Pleae follow the existing style of the script.
```suggestion
    local docker_args=()
```